jbmc/src/janalyzer/janalyzer_parse_options.cpp

```diff
@@ -733,7 +733,14 @@ void janalyzer_parse_optionst::help()
     "Usage:                       Purpose:\n"
     "\n"
     " janalyzer [-?] [-h] [--help] show help\n"
-    " janalyzer class              name of class to be checked\n"
+    " janalyzer class              name of class or JAR file to be checked\n"
+    "                              In the case of a JAR file, if a main class"
+    " can be inferred\n"
```

Comment:
The help text should be broken after 80 chars and properly aligned.

Reply:
I think the formatting is correct now (as it was done below in the same file).